cpliakas/git-wrapper/3.0/config/packages/cpliakas_git_wrapper.yaml

```diff
@@ -1,2 +1,15 @@
 services:
     GitWrapper\GitWrapper: null
+
+# To add logging capabilities use monolog and this configuration
+# monolog:
```

Comment:
I would remove the space between # and the content so that people can just remove the comment and it works (probably even more important for the other lines to have the right number of spaces).